cql2pgjson/src/test/resources/foreignKey.sql

```diff
@@ -1,15 +1,18 @@
-DROP TABLE IF EXISTS tablea, tableb, tablec;
+DROP TABLE IF EXISTS tablea, tableb, tablec, tabled;
 CREATE EXTENSION IF NOT EXISTS unaccent WITH SCHEMA public;
 CREATE OR REPLACE FUNCTION f_unaccent(text) RETURNS text AS $$
   SELECT public.unaccent('public.unaccent', $1);
 $$ LANGUAGE sql IMMUTABLE;
-CREATE TABLE tablea (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
+CREATE TABLE tabled (id UUID PRIMARY KEY, jsonb JSONB NOT NULL);
+CREATE TABLE tablea (id UUID PRIMARY KEY, jsonb JSONB NOT NULL, tabledId UUID references tabled);
```

Comment:
This test setup intendes to have foreign key pointers from tablec -> tableb -> tablea.

For testing child -> parent foreign key we should use tableb and make a CQL query using tablea.*. Can you remove tablea.tabledId and tabled?